stdlib/tkinter/__init__.pyi

```diff
@@ -667,12 +670,24 @@ class Wm:
         self, minNumer: None = None, minDenom: None = None, maxNumer: None = None, maxDenom: None = None
     ) -> tuple[int, int, int, int] | None: ...
     aspect = wm_aspect
-    @overload
-    def wm_attributes(self) -> tuple[Any, ...]: ...
+    if sys.version_info >= (3, 13):
+        @overload
+        def wm_attributes(self, *, return_python_dict: Literal[False] = False) -> tuple[Any, ...]: ...
+        @overload
+        def wm_attributes(self, *, return_python_dict: Literal[True]) -> dict[str, Any]: ...
+
+    else:
+        @overload
+        def wm_attributes(self) -> tuple[Any, ...]: ...
+
     @overload
     def wm_attributes(self, option: str, /): ...
     @overload
     def wm_attributes(self, option: str, value, /, *__other_option_value_pairs: Any) -> None: ...
+    if sys.version_info >= (3, 13):
+        @overload
+        def wm_attributes(self, **kwargs: Any) -> None: ...
```

Comment:
Because it is possible to type the kwargs properly, it would be better to leave them unannotated (or use `_typeshed.Incomplete`). If you create a `TypedDict` for the return value, you can just fill in the same kwargs here :)
```suggestion
        def wm_attributes(self, **kwargs) -> None: ...  # TODO: type the kwargs
```